dbclient/jdbc/src/main/java/io/helidon/dbclient/jdbc/JdbcConnectionPool.java

```diff
@@ -97,64 +100,54 @@ default String dbType() {
         return JdbcClientProvider.JDBC_DB_NAME;
     }
 
+    @Override
+    default String name() {
```

Comment:
If you want to keept this as a functional interface, this should not throw an exception. Please replace with `inlined` (or similar) string as the name and type.

Reply:
But why? An exception will force user of this interface to override it. Common name will not.